src/test/java/org/example/TimeZoneConverterTest.java

```diff
@@ -0,0 +1,47 @@
+package org.example;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TimeZoneConverterTest {
+
+    @Test
+    public void testConvertInstantToNewYork() {
+        LocalDateTime fixedDateTime = LocalDateTime.parse("D/2024-08-08 T/05:15:00");
+        Instant fixedInstant = fixedDateTime.atZone(ZoneId.systemDefault()).toInstant();
+        LocalDateTime expected = fixedInstant.atZone(ZoneId.of("America/New_York")).toLocalDateTime();
```

Comment:
Do not use your test input when you are setting expected value.
Use constant values also here